File: paciente/src/main/java/com/sevenine/conecta/repository/entity/EstadoCivilEntity.java
package com.sevenine.conecta.repository.entity;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "estado_civil")
@AllArgsConstructor
@Builder
@Getter
@Setter
@NoArgsConstructor
public class EstadoCivilEntity {

    @Id
    @Column(name = "id")
    private Long id;

    @Column(name = "tipo", length = 10)
    private String tipo;

    @Column(name = "descricao", length = 255)
    private String descricao;

}

File: contato/src/main/java/com/sevenine/conecta/adapter/output/entity/EstadoCivilEntity.kt
package com.sevenine.conecta.adapter.output.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Table(name = "estado_civil")
@Entity(name = "EstadoCivil")
data class EstadoCivilEntity(
        @Id
        val id: Long,
        val tipo: String,
        val descricao: String,
)
